README.md

```diff
@@ -1 +1,46 @@
 # rancher-desktop-networking
+Is a simple layer 2 switch that allows connectivity between the VM and the host over `AF_VSOCK` written in Go using [gvisor's](https://github.com/google/gvisor) network stack and [gvisor-tap-vsock](https://github.com/google/gvisor). In addition to piping the ethernet frames, it also provides a configurable DNS, dynamic port forwarding and DHCP.
+
+Rancher Desktop Networking consists of three main components: host-switch, vm-switch, and network-setup. Below is a brief explanation of what each component does.
+
+## host-switch
+For now, it only runs on Windows and acts as a receiving end for all traffic that originated from the WSL VM (designated namespace). It performs a handshake to find the right VM to talk to over `AF_VSOCK`. Once a correct VM is found, it then listens for the incoming traffic from that VM. In additon to this, it can provide a configurable DNS resolver that runs in the user space network along with an API that allows for dynamic port forwarding.
```

Comment:
No `it can provide…` — it's always enabled, and hard coded with our host names. (There's nothing wrong with it being static, but the documentation needs to match the reality.)